sanic/sanic.py

```diff
@@ -71,7 +74,7 @@ def register_middleware(middleware):
             if attach_to == 'request':
                 self.request_middleware.append(middleware)
             if attach_to == 'response':
-                self.response_middleware.append(middleware)
+                self.response_middleware.insert(0, middleware)
```

Comment:
Why not use a [deque](https://docs.python.org/2/library/collections.html?highlight=collections#collections.deque)?